client/src/app/tasks/tasks.component.spec.ts

```diff
@@ -1,19 +1,18 @@
 import { CommonModule } from '@angular/common';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
-import { Store, StoreModule } from '@ngrx/store';
+import { of } from 'rxjs';
 import { CoreModule } from '../core/core.module';
 import { SharedModule } from '../shared/shared.module';
-import * as TasksAction from './shared/tasks.actions';
+import { TaskFacade } from './shared/task.facade';
 import { Tasks } from './shared/tasks.model';
-import { tasksReducer } from './shared/tasks.reducer';
 
 import { TasksComponent } from './tasks.component';
 
 describe('TasksComponent', () => {
   let component: TasksComponent;
   let fixture: ComponentFixture<TasksComponent>;
-  let store: Store<Tasks>;
+  let taskFacadeSpy = jasmine.createSpyObj('TaskFacade', ['getList']);
```

Comment:
__[TSLint]__ Identifier 'taskFacadeSpy' is never reassigned; use 'const' instead of 'let'. (prefer-const) ([link](https://sider.review/gh/repos/72339652/pulls/68#issue-4037364))

<sub>posted by Sider</sub>